wire/internal/wire/parse.go

```diff
@@ -845,7 +845,7 @@ func (pv ProviderOrValue) Provider() *Provider {
 	return pv.p
 }
 
-// Provider returns pv as a Value pointer. It panics if pv points to a
+// Value returns pv as a Value pointer. It panics if pv points to a
```

Comment:
I believe this is already fixed on master, can you merge?

Reply:
Done.